source/macos/AdaptiveCards/AdaptiveCards/Renderers/AdaptiveCardRenderer.swift

```diff
@@ -1,6 +1,18 @@
 import AdaptiveCards_bridge
 import AppKit
 
+class HeightTypeSupport {
+    private let supportElemets: [ACSCardElementType] = [.textBlock, .container]
```

Comment:
this will have entry for column later ?

Reply:
Yes, we are currently wrapping as much as we can. That is why I include this. It will be removed later, once the majority of the work is completed.